lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text"><%- gettext("Sign in") %></span>
+        </h2>
+    </div>
+
+    <p><%- _.sprintf( gettext("Would you like to sign in using your %(providerName)s credentials?"), { providerName: hintedProvider.name } ) %></p>
+
+    <button class="action action-primary action-update proceed-button button-<%- hintedProvider.id %> hinted-login-<%- hintedProvider.id %>">
+        <div class="icon fa <%- hintedProvider.iconClass %>" aria-hidden="true"></div>
```

Comment:
FYI the default is http://fortawesome.github.io/Font-Awesome/icon/sign-in/ - e.g. see Dummy in screenshot below. Is that ok, or do you really prefer no icon at all? 

![screen shot 2015-06-25 at 3 03 40 pm](https://cloud.githubusercontent.com/assets/945577/8366627/6cc2322e-1b4b-11e5-8cd5-ec920576fe3f.png)

*or*

![screen shot 2015-06-25 at 3 05 48 pm](https://cloud.githubusercontent.com/assets/945577/8366669/b208994a-1b4b-11e5-8fea-88b7d051c2bb.png)